Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml

```diff
@@ -75,10 +75,7 @@ configuration:
   name: incidentType
   required: false
   type: 13
-- additionalinfo: |-
```

Comment:
Why did you remove the additional info?

Reply:
I think we are not using this key anymore cause it made sdk-validate fail on scheme error.